Android Exoplayer - Failed

## Android ExoPlayer - Build Execution Fails

I am trying to run Android Build with ExoPlayer and I am getting the following error.

````
error Failed to install the app. Make sure you have the Android development environment set up: https://reactnative.dev/docs/environment-setup.
Error: Command failed: ./gradlew app:installDebug -PreactNativeDevServerPort=8081
/Users/mohibwasay/Desktop/workspace/relevnt/relevnt-app/node_modules/react-native-video/android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java:330: error: cannot find symbol
        ImageButton playButton = playerControlView.findViewById(R.id.exo_play);
                                                                    ^
  symbol:   variable exo_play
  location: class id
/Users/mohibwasay/Desktop/workspace/relevnt/relevnt-app/node_modules/react-native-video/android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java:342: error: cannot find symbol
        ImageButton pauseButton = playerControlView.findViewById(R.id.exo_pause);
                                                                     ^
````

#### build.gradle

````
plugins {
    id "com.android.application" version '7.1.2' apply false
    id "com.android.library" version '7.1.2' apply false
    id "org.jetbrains.kotlin.android" version "1.6.10" apply false
    id "com.google.gms.google-services" version "4.3.10" apply false
    id "com.osacky.doctor" version "0.8.0"
}

ext {
    buildToolsVersion = "33.0.0-rc2"
    minSdkVersion = 21
    compileSdkVersion = 31
    targetSdkVersion = 30
    supportLibVersion = "28.0.0"
    firebaseBomVersion = "28.2.0"
    googlePlayServicesVersion = "18.+"
    ndkVersion = "21.4.7075529"
    googlePlayServicesAuthVersion = "19.2.0" 
    kotlin_version = "1.6.10"
}


task clean(type: Delete) {
    delete rootProject.buildDir
}

````

--- a/android/src/main/res/layout/exo_player_control_view.xml
+++ b/android/src/main/res/layout/exo_player_control_view.xml
@@ -14,27 +14,27 @@
         android:paddingTop="4dp"
         android:orientation="horizontal">
 
-        <ImageButton android:id="@id/exo_prev"
+        <ImageButton android:id="@+id/exo_prev"
             style="@style/ExoMediaButton.Previous"/>
 
-        <ImageButton android:id="@id/exo_rew"
+        <ImageButton android:id="@+id/exo_rew"
             style="@style/ExoMediaButton.Rewind"/>
         <FrameLayout
             android:id="@+id/exo_play_pause_container"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_gravity="center">
-            <ImageButton android:id="@id/exo_play"
+            <ImageButton android:id="@+id/exo_play"
                 style="@style/ExoMediaButton.Play"/>
 
-            <ImageButton android:id="@id/exo_pause"
+            <ImageButton android:id="@+id/exo_pause"
                 style="@style/ExoMediaButton.Pause"/>
         </FrameLayout>
 
-        <ImageButton android:id="@id/exo_ffwd"
+        <ImageButton android:id="@+id/exo_ffwd"
             style="@style/ExoMediaButton.FastForward"/>
 
-        <ImageButton android:id="@id/exo_next"
+        <ImageButton android:id="@+id/exo_next"
             style="@style/ExoMediaButton.Next"/>
 
     </LinearLayout>
@@ -46,7 +46,7 @@
         android:gravity="center_vertical"
         android:orientation="horizontal">
 
-        <TextView android:id="@id/exo_position"
+        <TextView android:id="@+id/exo_position"
             android:layout_width="50dp"
             android:layout_height="wrap_content"
             android:textSize="14sp"
@@ -57,12 +57,12 @@
             android:textColor="#FFBEBEBE"/>
 
         <com.google.android.exoplayer2.ui.DefaultTimeBar
-            android:id="@id/exo_progress"
+            android:id="@+id/exo_progress"
             android:layout_width="0dp"
             android:layout_weight="1"
             android:layout_height="26dp"/>
 
-        <TextView android:id="@id/exo_duration"
+        <TextView android:id="@+id/exo_duration"
             android:layout_width="50dp"
             android:layout_height="wrap_content"
             android:textSize="14sp"
@@ -73,7 +73,7 @@
             android:textColor="#FFBEBEBE"/>
 
         <ImageButton
-            android:id="@id/exo_fullscreen"
+            android:id="@+id/exo_fullscreen"
             style="@style/ExoMediaButton.FullScreen"
             android:layout_width="30dp"
             android:layout_height="30dp"

